### DISCUSSION TITLE
Creating a new branch from a tag in the github website

### DISCUSSION THREAD

**Discussion by MithunKinarullathil** [community/community#19259] [Other Feature Feedback, Questions, & Ideas] [answered]
Does anyone know if it is possible? Currently when I go to the branches page of the repository and click on the new branch Icon I can only select the existing branch as the source but not a tag. Does anyone know if this can be done?
![image](https://user-images.githubusercontent.com/51120171/175257792-9c3ca54e-293c-4dcf-9bd8-b4c2853ee967.png)

**Comment by bn-afournier** [comment]
I was looking for the same feature.  We're migrating from Bitbucket to GitHub; and the former provides this option in the web UI.  It's not used super-often, but it was a helpful feature.

They only alternative may be to create the branch using the git cli and then push the new branch to the remote server (as discussed [elsewhere](https://gist.github.com/nickfloyd/798119)).

**Comment by MithunKinarullathil** [comment]
Zi-Tao had mentioned a good solution. We do have an option to create branches from tag here on the repository branches button.
![image](https://user-images.githubusercontent.com/51120171/176866888-eb776931-c665-4844-868b-b53bd186fda2.png)

**Comment by Zi-Tao** [comment, marked as answer]
@MithunKinarullathil https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository
